src/documentation/docs/Sections/InterestingFactsSection.mdx

```diff
@@ -7,43 +7,48 @@ import PropsTableRow from "@/components/internal/Documentation/PropsTable/compon
 import SlotsTableRow from "@/components/internal/Documentation/SlotsTable/components/SlotsTableRow";
 
 <Meta
-  title="Interesting Facts Section"
-  subtitle="a complete interesting facts section"
+  title="InterestingFacts Section"
+  subtitle="Allows to display a text together with a background image and numerous counters."
 />
 
 ## Properties
 
 <PropsTable>
       <PropsTableRow value="headline" type="String" required>
-          The title to display with this InterestingFactsSection.
+          The title of this section, serves for marking the content above the text.
```

Comment:
What is intendet here? "marking" sounds odd to me. Why not simply "The section's title."?